### DISCUSSION TITLE
[v1.0] Table: addAction() on RelationManager results on Public method [myAction] not found on component: [filament.resources.relation-manager.list-records]

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#841] [Help]
Hello,

Dont't know if this is a bug or I'm doing something wrong here.

I'm trying to add an action on a table column on a RelationManager. I simply call `->action("myAction")` on the `Table` object  on my RelationManager class (in this case, `LineasRelationManager.php`).

Then I add the method to the RelationManager:

```php
    public function verPrecios(int $lineaId): void
    {
        $this->emit('openModal', 'articulos.view-detalle-precios-linea', [$lineaId]);
    }
```

I think this is supposed to work, but when clicking on any column field, i get an exception:

```
Livewire\Exceptions\MethodNotFoundException
Unable to call component method. Public method [verPrecios] not found on component: [filament.resources.relation-manager.list-records] 
```

I tried also to add the action method on my `EditRecord` component.

Is there something I'm missing?

**Comment by danharrin** [comment]
Hi, we don't support adding actions and further customisation of relation managers in v1 - as the exception explains, the method needs to be added to an internal LW component, not the relation manager itself.

Please upgrade to v2 when this is much easier - if you need support please find us on Discord :)
